sst/sstable.go

```diff
@@ -3,14 +3,16 @@ package sst
 import (
 	"bytes"
 	"encoding/binary"
+	"math"
```

Comment:
no need to re-order imports

Reply:
regarding order of imports - I just use  goland and default settings - can you configure your IDE to not re-order imports on save of file? (I know this is possible with goland, don't know about your IDE)